docs/user/alerting/alert-types.asciidoc

```diff
@@ -1,6 +1,6 @@
 [role="xpack"]
 [[alert-types]]
-== Alert types
+== Standard stack alert types
```

Comment:
Is this rename really necessary?

As a user, the word "standard" does not provide a lot of meaning and obscures the real contents "alert types". I would recommend leaving this title as is. I think its ok having a title "alert types" and "geo alert types"

Reply:
I'll have to think about this one. I'm open to changing the wording, but it seems strange to me to have a top-level category called "Alert types" and then have another called "Geo alert types". I chose "standard" since these alert types are included at the basic license level and aren't hidden behind a feature flag.